src/invertColor.js

```diff
@@ -13,7 +13,27 @@
  * @return {string}
  */
 function invertColor(color) {
-  // write code here
+  const r = '0x' + String(color).split('').splice(1, 2).join('');
```

Comment:
Ты вроде и так получаешь строку
```suggestion
  const r = '0x' + color.split('').splice(1, 2).join('');
```